redisConnections/redisConnection.py

```diff
@@ -42,12 +43,12 @@ def __init__(
         self._redis_host = config.get('DB-Section', 'redis-host')
         self._redis_port = int(config.get('DB-Section', 'redis-port'))
         if modules_db is None:
-            modules_db = config.get('DB-Section', 'redis-modules-db', fallback=1)
+            modules_db = config.get('DB-Section', 'redis-modules-db', fallback=RedisEnum.MODULES.value)
```

Comment:
Using two separate config files here is redundant. `redis-modules-db` never got added to the sample config or to the config in PROD afaik, so I think we should just get rid of these `config.get`s and just use the `RedisEnum` values directly.

Reply:
I think they can be used in some tests, cause we have them in test.conf for backend:
```
redis-modules-db=11
redis-vendors-db=14
redis-users-db=12
```

Reply:
Could these tests be changed to use the `TEST_MODULES`, etc. databases instead?

Reply:
Yeah, I think I can change it right now

Reply:
Actually, I checked and I am now not sure if we want to do this, because we have tests that are not using RedisConnection or RedisUsersConnection directly (these are the only classes that use redis-modules-db, redis-vendors-db and redis-users-db), but they are testing other code, that uses default variant of this classes and basically if we want to remove those config.get()'s, then we need to mock every such test (we need to add test parameters for every default RedisConnection and RedisUsersConnection)
So, do we still want to do this?

Reply:
I think we should do something similar to what we do with the .conf file. So have a file in `/var/yang/redis` or whatever with the real values, and have a file in `tests/resources` with test values.
So we won't need a `TEST_MODULES` entry, we'll just use the `MODULES` entry in the test config while testing and the `MODULES` entry in the real config when actually running. Does that make sense?

Reply:
Yeah, totally, that will work